static/styles/portico-signin.scss

```diff
@@ -289,7 +289,7 @@ html {
 }
 
 .register-form button {
-    margin-top: 22px;
+    margin-top: 25px;
 }
```

Comment:
This also moves around the buttons on http://localhost:9991/register/; I think we want to adjust change something more focused here.